src/translations.h

```diff
@@ -35,7 +35,13 @@ const char *npgettext( const char *context, const char *msgid, const char *msgid
 
 const char *strip_positional_formatting( const char *msgid );
 
+// If PRINTF_CHECKS is enabled, have to skip the function call so GCC can see the format string
+#if defined(PRINTF_CHECKS) && defined(__GNUC__)
```

Comment:
`make clean` fixed the problem indeed, I should have checked that first, thank you!